src/psl/psl-node.c

```diff
@@ -69,18 +69,26 @@ static const imask_t has_map[P_LAST_PSL_KIND] = {
    (I_SUBKIND | I_VALUE),
 
    // P_SERE
-   (I_PARAMS | I_CLOCK),
+   (I_SUBKIND | I_PARAMS | I_CLOCK | I_REPEAT | I_DECLS),
```

Comment:
The `I_SUBKIND` isn't used any more?

Reply:
That is true at the moment. As the very next step, I want to add parsing of all SERE kinds (`Compound_SERE`, `SERE concatenation` and `SERE fusion`), so It will be there. I already have it locally, just not pushed yet. I can remove it for this MR if you want, but It will come soon again.